src/services/amm.ts

```diff
@@ -8,8 +9,10 @@ import { mainnetTxAssembler } from './defaultTxAssembler';
 import { explorer } from './explorer';
 import yoroiProver from './yoroi/prover';
 
-export const ammOrderRefunds = new AmmOrderRefunds(
-  explorer,
+export const ergopayAmmOrderRefunds = new AmmOrderRefunds(explorer);
+
+export const ammOrderRefunds = new AmmOrderRefundsWrapper(
+  ergopayAmmOrderRefunds,
```

Comment:
is it normal to have "ergo" in this file? it is services/amm for all networks as I understand